ush/global_postevents.sh

```diff
@@ -94,20 +94,16 @@ set -u
 TIMEIT=${TIMEIT:-""}
 [ -s $DATA/time ] && TIMEIT="$DATA/time -p"
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
+sys_tp=${sys_tp:-$(getsystem -tp)}
```

Comment:
`getsystem -tp` is not valid on WC2. You need to use different options or none at all. I hashed those out in an email a few days ago that you may want to review.

Reply:
doh! cleaned defunct -tp